test/functional/ex_cmds/quickfix_commands_spec.lua

```diff
@@ -47,6 +47,7 @@ for _, c in ipairs({'l', 'c'}) do
       eq(('%s-2.res'):format(file), funcs.bufname(list[2].bufnr))
 
       -- Run cfile/lfile from a modified buffer
+      command('set nohidden')
       command('enew!')
       curbufmeths.set_lines(1, 1, true, {'Quickfix'})
       eq(('Vim(%s):E37: No write since last change (add ! to override)'):format(
```

Comment:
Need to set 'nohidden' to trigger this error message, though I'm not sure what exactly this is supposed to be testing.